Sources/HummingbirdCore/Server/HTTP/HTTP1Channel.swift

```diff
@@ -22,15 +22,48 @@ import NIOHTTPTypesHTTP1
 public struct HTTP1Channel: ServerChildChannel, HTTPChannelHandler {
     public typealias Value = NIOAsyncChannel<HTTPRequestPart, HTTPResponsePart>
 
+    /// HTTP1Channel configuration
+    public struct Configuration: Sendable {
+        /// Additional channel handlers to add to channel
+        public var additionalChannelHandlers: @Sendable () -> [any RemovableChannelHandler]
```

Comment:
Actually: Maybe this property can be clarified to reflect the position of the handlers?

Reply:
Fixed comment